packages/primes/primes.1.0/opam

```diff
@@ -0,0 +1,30 @@
+
+opam-version: "1.2"
+name: "primes"
+version: "1.0"
+maintainer: "Kit Freddura <kitfreddura@gmail.com>"
+authors: "Kit Freddura <kitfreddura@gmail.com>"
+license: "MIT License"
+dev-repo: "https://github.com/KitFreddura/OCaml-Primes.git"
+homepage: "https://github.com/KitFreddura/OCaml-Primes"
+bug-reports: "None yet"
+build: [
+  ["mkdir" "%{prefix}%/lib/primes"]
+]
+ocaml-version: [>= "4.01"]
+install: [
+["ocamlfind" "ocamlc" "-c" "-linkpkg" "-package" "zarith" "primes.mli"]
+["ocamlfind" "ocamlc" "-thread" "-o" "primes.cma" "-linkpkg" "-package" "zarith,core,gen" "primes.ml"]
+["cp" "primes.cmo" "%{prefix}%/lib/primes/"]
+["cp" "primes.cmi" "%{prefix}%/lib/primes/"]
+["cp" "primes.cma" "%{prefix}%/lib/primes/"]
+["cp" "primes.mli" "%{prefix}%/lib/primes/"]
+["cp" "META" "%{prefix}%/lib/primes/"]
+]
```

Comment:
Have you considered using [`ocamlfind`'s `install` command](http://projects.camlcity.org/projects/dl/findlib-1.2.1/doc/guide-html/c348.html#AEN350)?

It should be possible to replace these five lines with a single call to `ocamlfind install ...`